recipes-iot/aws-iot-securetunneling-localproxy/aws-iot-securetunneling-localproxy_git.bb

```diff
@@ -8,8 +8,10 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
 
 DEPENDS += "boost catch2 openssl protobuf protobuf-native zlib"
 
+BRANCH ?= "master"
+
 PV = "1.0+git${SRCPV}"
-SRC_URI = "git://git@github.com/aws-samples/aws-iot-securetunneling-localproxy.git;protocol=ssh"
+SRC_URI = "git://git@github.com/aws-samples/aws-iot-securetunneling-localproxy.git;branch=${BRANCH};protocol=ssh"
```

Comment:
Why SSH Protocol?

Reply:
Oh I didn't even see that. I didn't change it. I only changed what was required.